README.md

```diff
@@ -65,6 +65,7 @@ Check ```com.infusion.relnotesgen.MainITTest``` for appropriate use case.
 | issue.sort.type			| -issueSortType 				| Defines sort order of issues by type - issues are provided in report template context as map where key is issue type and value is list of issues with that type, this map is sorted | New Feature,Bug |
 | issue.sort.priority		| -issueSortPriority 			| In report template issues are provided as map where key is issue type and value is list of issue with that type, this parameter defines order in list of issues | Highest,High,Medium,Low,Lowest |
 | report.directory			| -reportDirectory 				| Directory where release notes will be saved | C:/temp |
+| report.clientFacingFilters			| -clientFacingFilters 				| Indicates fields which indicate issue should be included in client facing report | Requirement VA ID,Defect_Id |
```

Comment:
I'm getting this:
`releasenotesreport git:(client-facing-notes) ✗ java -jar target/release-notes-generator-1.0-SNAPSHOT.jar -configurationFilePath ../configuration.properties -commitId2 5545fddf8e5cab87ad33784e66a4c39f00425860 -commitId1 a002685c459d6c4e24e2905211fea1fdae57d702 -jiraFixVersions "R3.3 - Harmony 1.4.0_?" -clientFacingFilters
15:00:22.097 INFO  c.i.r.l.ReleaseNotesLogger - Reading program parameters...
Exception in thread "main" com.beust.jcommander.ParameterException: Unknown option: -clientFacingFilters`